ldes-server-infra-mongo/mongo-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/member/MemberMongoRepository.java

```diff
@@ -55,7 +59,8 @@ public void deleteMember(String memberId) {
 
 	@Override
 	public void deleteMembersByCollection(String collection) {
-		repository.deleteAllByCollectionName(collection);
+		Long deleteCount = repository.deleteAllByCollectionName(collection);
+		log.info("Deleted member count: {}", deleteCount);
```

Comment:
True.. at first I was too lazy to use a placeholder and used string concatenation but then changed it because of sonar and it didn't occur to me, to write it more natural :)